apis/infrastructure/v1beta1/byocluster_webhook.go

```diff
@@ -63,10 +61,5 @@ func (byoCluster *ByoCluster) ValidateDelete() error {
 }
 
 func validateByoClusterSpec(spec ByoClusterSpec) field.ErrorList {
```

Comment:
This webhook is pretty much non-functional and empty. Do we even need this anymore?